### PR TITLE
Updated CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by pramod74**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design

/kind documentation
> /kind feature
> /kind release

**What this PR does / why we need it**:
Added documentation on requiring signed commits in CONTRIBUTING.md

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., usage docs, etc.**:

```docs

```

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Contributing to MagTape
+	# Contributing to MagTape
```
Comment (phenixblue) [inline CONTRIBUTING.md:1]: nitpick(blocking): I don't think this extra whitespace is needed

**Review by phenixblue — CHANGES_REQUESTED** [review]
Looks good other than the extra whitespace added before the title

File: CONTRIBUTING.md (hunk 1 above)
Comment (pramod74) [reply CONTRIBUTING.md]: Thanks, @phenixblue for reviewing my first submission.
I have added another commit to my PR after removing the leading whitespace from the title. Kindly review.

**Review by phenixblue — APPROVED** [review]
/lgtm

**Comment by pramod74** [comment]
> /lgtm

Thanks!

**Pull request merged**